src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/service/EcrcServicesImpl.java

```diff
@@ -45,24 +36,14 @@
 	@Autowired
 	private EcrcProperties ecrcProps;
 
-	private WebClient webClient = null;
-
-	private final Logger logger = LoggerFactory.getLogger(EcrcServicesImpl.class);
-
 	@Autowired
-	ObjectMapper objectMapper;
-
-	@PostConstruct
-	public void InitService() {
+	private EcrcWebMethodsServiceImpl ecrcWebMethodsService;
```

Comment:
Clarification: Is there a reason why EcrcWebMethodsServiceImpl is instantiated instead of EcrcWebMethodsService?

Reply:
Good catch it should not be the implementation. I have updated the code.